x-pack/plugins/reporting/common/types/export_types/csv_saved_object.ts

```diff
@@ -5,10 +5,14 @@
  * 2.0.
  */
 
+import { estypes } from '@elastic/elasticsearch';
 import type { BaseParams, BasePayload } from '../base';
 
 interface CsvFromSavedObjectBase {
   objectType: 'saved search';
+  state?: {
+    query: estypes.QueryDslQueryContainer | estypes.QueryDslQueryContainer[];
```

Comment:
NOTE: `sort` and `docvalue_fields` are still not supported, [as they had been in the older, removed code](https://github.com/elastic/kibana/pull/34571/files#diff-0af8edc988cc05542af815e98c0b4e63da0c1e68cfaa92faca56bbf8e6c17ffcR87-R90).